### PR TITLE
Allow-Cloudwatch-logs-to-not-be-used

### PR DESCRIPTION
**Pull request by cahnk**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -16,6 +16,12 @@ variable "enable_log_file_validation" {
   default     = true
 }
 
+variable "enable_logging" {
+  description = "Specifies whether to enable CloudWatch logging if it is configured"
```
Comment (lorengordon) [inline variables.tf:20]: This variable simply enables or disables logging for the _trail_. It is not specific to CloudWatch.

File: tests/no_log_group/main.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+module "baseline" {
+  source = "../../"
+
+  create_kms_key             = false
+  cloudtrail_name            = random_id.name.hex
+  cloudtrail_bucket          = aws_s3_bucket.this.id
+  use_existing_log_group     = false
+  create_log_group           = false
+  enable_log_file_validation = false
+  enable_logging             = false
```
Comment (lorengordon) [inline tests/no_log_group/main.tf:32]: Setting this to `false` does not not control whether the cloudwatch log group is created. It simply stops the trail from recording any events at all.

**Review by lorengordon — COMMENTED** [review]
I kinda feel like we should be able to accomplish this with a single new variable, instead of two. (The third variable in this patch is just exposing a new argument to cloudtrail, not specific to cloudwatch logs, which is fine.)

File: tests/no_log_group/main.tf (hunk 2 above)
Comment (cahnk) [reply tests/no_log_group/main.tf:31]: Concur.  This variable is just an on/off setting for logging.  The CW log group would have to preexist in order for it to be enabled.  This is actually just one of the Terraform aws_cloudtrail arguments.  

enable_logging - (Optional) Enables logging for the trail. Defaults to true. Setting this to false will pause logging.

I am just exposing it for use by our module.

File: variables.tf (hunk 1 above)
Comment (cahnk) [reply variables.tf:20]: Concur.  I removed the word CloudTrail from the description.

**Review by lorengordon — APPROVED** [review]

**Pull request merged**